sdk/eventhubs/azure-messaging-eventhubs/CMakeLists.txt

```diff
@@ -90,7 +89,7 @@ set(
     src/producer_client.cpp
     src/retry_operation.cpp
     src/eventhubs_utilities.cpp
-)
+ "src/processor.cpp")
```

Comment:
Why the quote?  Can you fix alignment

Reply:
Because visual studio loves to put quotes everywhere and has no concept of alignment.